puffin_http/src/server.rs

```diff
@@ -53,7 +53,7 @@ impl Server {
                     if let Err(err) = server_impl.accept_new_clients() {
                         log::warn!("puffin server failure: {}", err);
                     }
-                    if let Err(err) = server_impl.send(&*frame) {
+                    if let Err(err) = server_impl.send(&frame) {
```

Comment:
What is the clippy error/warning fixed by this change ?

Reply:
Ah im used that our CI automatically does that. Good to know in the future.

Reply:
Ak, I had not seen that the merge had been done automatically.